pandas/core/arrays/base.py

```diff
@@ -539,13 +540,14 @@ def shift(self, periods: int = 1, fill_value: object = None) -> ABCExtensionArra
 
         fill_value : object, optional
             The scalar value to use for newly introduced missing values.
-            The default is ``self.dtype.na_value``
+            The default is ``self.dtype.na_value``.
 
             .. versionadded:: 0.24.0
 
         Returns
         -------
-        shifted : ExtensionArray
+            ExtensionArray
```

Comment:
I think this needs to be unindented to render properly

Reply:
agreed, thanks

Reply:
@HughKelley can you fix this please, so we can merge. Thanks!

Reply:
@datapythonista should I make the change with an additional commit or is there another way to do this?

Reply:
Additional commit.

Reply:
Can you fix the indantation here @HughKelley please?